cookbook/integrations/kubernetes/ray_example/ray_example.py

```diff
@@ -0,0 +1,74 @@
+"""
+Ray tasks
+-------------------------------
+Ray task allow you to run a Ray job on existing Ray cluster or create a Ray cluster by using Ray operator.
+
+
+Let's get started with an example!
+"""
+
+# %%
+# First, we load the libraries.
+import typing
+
+import ray
+from flytekit import Resources, task, workflow
+from flytekitplugins.ray import HeadNodeConfig, RayJobConfig, WorkerNodeConfig
+
+
+# %%
+# Ray task
+# ========
+#
+# We define a ray_example `remote function <https://docs.ray.io/en/latest/ray-core/tasks.html#tasks>`__ that will be executed asynchronously in the Ray cluster.
+@ray.remote
+def f(x):
+    return x * x
+
+
+# %%
+# Defining an Ray config
+# ====================
+#
+# We create a HeadNodeConfig and WorkerNodeConfig for the Ray job, and these config will be used by Ray operator to launch a Ray cluster before running the task.
+#
+# * ``ray_start_params``: `RayStartParams <https://docs.ray.io/en/latest/ray-core/package-ref.html#ray-start>`__ are the params of the start command: address, object-store-memory
+# * ``replicas``: Desired replicas of the worker group. Defaults to 1.
+# * ``group_name``: RayCluster can have multiple worker groups, and it distinguishes them by name
+# * ``runtime_env``: A `runtime environment <https://docs.ray.io/en/latest/ray-core/handling-dependencies.html#runtime-environments>`__ describes the dependencies your Ray application needs to run, and it's installed dynamically on the cluster at runtime.
+#
+ray_config = RayJobConfig(
+    head_node_config=HeadNodeConfig(ray_start_params={"log-color": "True"}),
+    worker_node_config=[WorkerNodeConfig(group_name="ray-group", replicas=5)],
+    runtime_env={"pip": ["numpy", "pandas"]},
```

Comment:
fine for now, would like to move to an example where a requirements file is given.